WebPhapp/backend/server.js

```diff
@@ -24,47 +49,125 @@ app.get('/api/v1/list', (req,res) => {
     console.log('Sent list of items');
```

Comment:
Again, can we add logging instead of just console.logs? I think have different debugging levels would be really useful .

Reply:
We are not yet close to running on a production level. In the immediate term, what benefit is there to adding more complicated logging? I am currently looking at https://www.npmjs.com/package/winston, but I don't yet see it as worth the effort. I can be convinced.

Reply:
Within our `ProbSyllab` project I have always enjoyed the ability to change the logging levels to see what I need. So, I thought having the same setup would be worthwhile.